h/streamer/streamer.py

```diff
@@ -20,6 +20,11 @@
 ANNOTATION_TOPIC = "annotation"
 USER_TOPIC = "user"
 
+TOPIC_HANDLERS = {
+    ANNOTATION_TOPIC: messages.handle_annotation_event,
+    USER_TOPIC: messages.handle_user_event,
+}
```

Comment:
Pull this out so we can directly compare it in the tests instead of some monstrosity we create there.